src/lib/components/snackbar/snackbar.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Snackbar from './snackbar.js';
+
+describe('Snackbar', () => {
```

Comment:
Can we have more tests, one for each type, and one for `message` and another one for `children`? :)